### ISSUE TITLE
Invoking issues after recent update

### ISSUE THREAD

**Issue by mattszymanko** [FINDarkside/TLD-Developer-Console#16]
Hey,
I have unfortunately faced some issues with invoking the console after the recent update - Specifically, I was unable to do so.
On top of that, an error has occured, as expected. Could probably use some further investigating.
```UnhollowerBaseLib.Il2CppException: System.ArgumentException: The Object you want to instantiate is null.
  at UnhollowerBaseLib.Il2CppException.RaiseExceptionIfNecessary (System.IntPtr returnedException) [0x00018] in <6f5e4ff5bbd94258ab8bd773251f8e6c>:0
  at UnityEngine.Object.Instantiate (UnityEngine.Object original) [0x00034] in <9171f274066e44e595e26a664b2ecc47>:0
  at DeveloperConsole.InitializePatch.Prefix () [0x0000b] in <1e442c6e746a4765b1efd23b1a600f8d>:0
  at (wrapper dynamic-method) BootUpdate.DMD<BootUpdate::Start>(BootUpdate)
  at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.DMD<BootUpdate::Start>_il2cpp(intptr)
```

Sincerely, Matt

**Comment by zeobviouslyfakeacc** [comment]
Hi @mattszymanko 

We're well aware of these issues. It's been only a day since the update, and most mods are broken due to the large-scale changes in TLD.

Give it some time, we'll be back soon 🙂

**Comment by mattszymanko** [comment]
Hello again,
Indeed, no worries, I'm perfectly aware of the fact that the update has just released. However, I'm just making sure you're well informed, since the issue is technically there either way. You know how it is.
Thanks for the response, appreciate it.

Matt

**Comment by RubinsJ** [comment]
Hello can I help to work with this issue.

**Comment by ds5678** [comment]
Resolved in #18
